src/fr/articles/veille-avril-mai-2025.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Veille #A11y Avril-Mai 2025"
+abstract: "Florilèges de ressources en #a11y, Avril-Mai 2025"
+date: "2025-06-05"
```

Comment:
> French version: date: "2025-06-**07**"
> English version: date: "2025-06-**05**"

Correct realease date will depend of what you planned but they should go at the same date ;)